cypress/integration/page.spec.js

```diff
@@ -171,7 +171,7 @@ Cypress.on('fail', (e) => {
 
 describe('', () => {
   beforeEach(() => {
-    if (failed) Cypress.runner.stop();
+    // if (failed) Cypress.runner.stop();
   });
```

Comment:
Why do you edit this?

Reply:
Because this was inconvenient for me when I was fixing tests. I uncomment.